2020.csv

```diff
@@ -7,6 +7,7 @@ Moscow Python Conf++ 2020,2020-03-27,2020-03-27,"Moscow, Russia",RUS,Infospace,,
 PyCon Italy,2020-04-02,2020-04-05,"Florence, Italy",ITA,Grand Hotel Mediterraneo,,,https://www.pycon.it/en/,,https://www.pycon.it/en/sponsor/
 PyCon US,2020-04-15,2020-04-23,"Pittsburgh, Pennsylvania, United States of America",USA,David L. Lawrence Convention Center,2019-11-22,2019-12-20,https://us.pycon.org/,https://us.pycon.org/2020/speaking/talks/,https://us.pycon.org/2020/sponsors/
 PyCon Turkey,2020-05-02,2020-05-03,"Istanbul, Turkey",TUR,"Albert Long Hall, Bogazici University",,2020-01-15,https://tr.pycon.org/,https://www.papercall.io/pycon-turkey-2020,
+PyCon CZ,2020-05-06,2020-07-06,"Ostrava, Czech Republic",CZE,Dolní oblast Vítkovice,2020-03-31,2020-03-31,https://cz.pycon.org/2020/,https://cz.pycon.org/2020/cfp/,https://cz.pycon.org/2020/sponsorship/
```

Comment:
Start/end dates are not correct. It seems that the PyCon CZ will be two months long :) I'm gonna open a PR.

Reply:
Huh. Good catch.